src/sentry/tsdb/snuba.py

```diff
@@ -213,6 +213,7 @@ def get_data(
         aggregation="count()",
         group_on_model=True,
         group_on_time=False,
+        snuba_filters=None,
```

Comment:
This also feels weird to me, although `get_data` isn't part of the abstraction so it might not be as much of a big deal.

Reply:
added this too